src/lib/holocene/filter-or-copy-buttons.svelte

```diff
@@ -18,7 +18,12 @@
     {#if filterable}
       <button on:click|preventDefault|stopPropagation={onFilter}>
         {#key filtered}
-          <Icon name={filtered ? 'filter-solid' : 'filter'} class="h-4 w-4" />
+          <Icon
+            name="filter"
+            class="h-4 w-4 rounded-sm {filtered
```

Comment:
Do we want the loading bar to show on top of these icons?

![screenshot](https://user-images.githubusercontent.com/15069288/195657747-48086f79-903c-45d1-ba68-72feaf62d993.png)